doc/api-hidden.rst

```diff
@@ -156,6 +335,94 @@
    plot.FacetGrid.set_ticks
    plot.FacetGrid.map
 
+   CFTimeIndex.all
```

Comment:
Not sure about these but perhaps they should be hidden until our Index API is finalized and public.

Reply:
without these, the methods are still visible from the `CFTimeIndex` page but the links are broken. We can probably push the decision of where to put them back until the index refactoring is done.

Reply:
Yes they can stay in api-hidden